src/OrchardCore.Modules/OrchardCore.Media/Services/MediaFileStore.cs

```diff
@@ -1,82 +1,93 @@
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Threading.Tasks;
-using OrchardCore.StorageProviders;
+using OrchardCore.FileStorage;
 
 namespace OrchardCore.Media.Services
 {
     public class MediaFileStore : IMediaFileStore
     {
         private readonly IFileStore _fileStore;
+        private readonly string _publicUrlBase;
 
-        public MediaFileStore(IFileStore fileStore)
+        public MediaFileStore(IFileStore fileStore, string publicUrlBase)
         {
             _fileStore = fileStore;
+            _publicUrlBase = publicUrlBase;
         }
 
-        public string Combine(params string[] paths)
+        public MediaFileStore(IFileStore fileStore)
         {
-            return _fileStore.Combine(paths);
+            _fileStore = fileStore;
         }
 
-        public Task<IEnumerable<IFile>> GetDirectoryContentAsync(string subpath = null)
```

Comment:
It's not really the scope of Azure storage but since already modified most of these methods could be expression-bodied.

Reply:
Indeed they could - the .editorconfig file in the repo doesn't contain a rule for this, so I wasn't sure what convention we use. Internally we (IDeliverable) use expression-bodied properties and indexers, but not methods. Is the Orchard Core convention to use expression bodies for everything?

Reply:
Since the solution already requires VS 2017 I see no harm in using everything up to C# 7(.1), especially that in some code files here you've already used expression-bodied properties.

Reply:
There's no technical reason, I guess it's more of a readability and coding convenience thing. Methods are more likely to have (or later get) more code I guess, making a block body a more stable choice. Personally I like that convention. But I am happy to adapt to whatever is currently the agreed-upon convention.

Reply:
That's fair enough - I like expression-bodied methods better, even if that means adding brackets at a later day, just as a matter of taste. So we should just agree on the taste most of us prefer, unless there are some other compelling reasons. We could have a quick poll at the meeting or something (even though I rarely attend the meeting nowadays :)).

Reply:
I recently watched a video about functional programming in C# (I'll paste the URL here when I find it again). The presenter said something about using expression bodies when returning a value (no side effects). I too think it's more elegant to use expression bodies, but we at IDeliverable decided to stick with the default setting (using brackets), assuming it would be the recommended default at large.